merl-an.opam

```diff
@@ -21,6 +21,11 @@ depends: [
   "ppx_fields_conv"
   "ppx_yojson_conv" {<= "v0.15.1"}
   "ptime"
+  "merlin-lib"
+  "dot-merlin-reader"
+  "merlin"
```

Comment:
Could you make merlin a test dependency (i.e. `with-test`)? Having merlin installed is not needed to compile `merl-an`.